aiohttp/test_utils.py

```diff
@@ -564,7 +565,7 @@ def timeout(*args, **kw):
                   protocol, payload_writer, task, loop,
                   client_max_size=client_max_size)
 
-    match_info = UrlMappingMatchInfo({}, mock.Mock())
+    match_info = UrlMappingMatchInfo({} if match_info is sentinel else match_info, mock.Mock())
```

Comment:
Isn't the line too long?

Reply:
didn't give any problems to me, waiting for the automated checks to pass

Reply:
You didn't run `make test`, travis blames exactly the line

Reply:
Yup, didn't run because was failing on my machine for some reason, that's why I was letting travis do the job.

In case you had this before, that's what I'm getting: `ValueError: option names {'--fast'} already added`

Reply:
uninstalling `pytest-aiohttp` worked for me